CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
 ## master
-
+* Added the ability to define a custom view controller for the top banner. ([#2121](https://github.com/mapbox/mapbox-navigation-ios/pull/2121))
```

Comment:
Mention the option property that serves as the entry point for this feature, since the public API is rather large for the developer to go hunting for it.